app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -38,28 +40,35 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
          * device. The result of the permission request is handled by a callback,
          * onRequestPermissionsResult.
          */
-            if (ContextCompat.checkSelfPermission(this.applicationContext,
-                            Manifest.permission.ACCESS_FINE_LOCATION)
-                    == PackageManager.PERMISSION_GRANTED) {
-                var locationPermissionGranted = true
-            } else {
-                val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION = 0
-                //ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
-                        //PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION)
-            }
+        if (ContextCompat.checkSelfPermission(
+                this.applicationContext,
+                Manifest.permission.ACCESS_FINE_LOCATION
+            )
+            == PackageManager.PERMISSION_GRANTED
+        ) {
+            var locationPermissionGranted = true
+        } else {
+            //Renamed because of underscores
+            val requestAccess = 0
```

Comment:
I'd keep this named as it were, because it will be used as a constant.   Why a constant?  Because this is a Unique ID passed in when permission is requested, and used to check which permission is granted.

But, refactor it and make it a field/attribute on the class itself, so that this one constant can be used in both places.